server/sonar-server/src/main/java/org/sonar/server/permission/ws/template/TemplateUsersAction.java

```diff
@@ -133,6 +137,7 @@ private static PermissionQuery buildQuery(Request wsRequest, PermissionTemplateD
         .addAllPermissions(permissionsByUserId.get(user.getId()));
       setNullable(user.getEmail(), userResponse::setEmail);
       setNullable(user.getName(), userResponse::setName);
```

Comment:
Existing email fields should be deprecated => Please update the JIRA ticket to say 'email' field is deprecated and add a changelog entry